pkg/util/util.go

```diff
@@ -207,6 +207,10 @@ func KubeCreateSkipExisting(obj runtime.Object) bool {
 		log.Printf("❌ Forbidden: %s %q: %s\n", gvk.Kind, objKey.Name, err)
 		return false
 	}
+	if errors.IsInvalid(err) {
```

Comment:
What does Invalid means - that the validations of the object failed, like missing a required field for example?

In which case did you get it? 
Is this is a static object that we create in the code? do we generally expect validation errors?

Reply:
we got it when the name of the bucket is illeagal: ```PANI[0004] ☠️  Panic Attack: [Invalid] ObjectBucketClaim.objectbucket.io "obc#1" is invalid: metadata.name: Invalid value: "obc#1"```